src/SplitPageLayout/SplitPageLayout.test.tsx

```diff
@@ -44,4 +44,30 @@ describe('SplitPageLayout', () => {
 
     expect(pane.getAttribute('id')).toBe('customId')
   })
+
+  it('renders Pane with a heading', () => {
+    const {getByText} = render(
+      <ThemeProvider>
+        <SplitPageLayout>
+          <SplitPageLayout.Pane heading="Custom heading" />
```

Comment:
Curious if it makes more sense to make the heading a component itself rather then passing it in as a prop:

```
          <SplitPageLayout.Pane.heading as='h3'>Custom heading"</SplitPageLayout.Pane.heading >
```


^ this would make it easier for people to add attributes in the future.